PukingPredator/Assets/Scripts/Eating.cs

```diff
@@ -93,20 +93,23 @@ private void GameInput_Puke()
         if (inventory.isEmpty) { return; }
 
         Consumable itemToPuke = inventory.PopItem();
-        itemToPuke.SetState(ItemState.beingPuked);
+        if (itemToPuke != null)

```

Comment:
make this a guard clause, no need to indent the entire function when you can just slap in
`if (itemToPuke == null) { return; }`